### PR TITLE
Fixes variable shadowing warnings

### PR DESCRIPTION
**Pull request by cchens**
Ref #613 

Fixes ghc's complaints about shadowing existing bindings, with some dubiously named variables. Also gets rid of some redundant imports.

### CODE REVIEW

File: hs/Database/CourseQueries.hs [hunk 1]
```diff
@@ -111,9 +111,9 @@ buildTutorial entity =
 -- | Builds a Session structure from a list of tuples from the Lectures table,
 -- and a list of tuples from the Tutorials table.
 buildSession :: [Entity Lectures] -> [Entity Tutorials] -> Maybe Tables.Session
-buildSession lectures tutorials =
```
Comment (david-yz-liu) [inline hs/Database/CourseQueries.hs]: The pluralization here makes sense. I would be okay with `lecs` and `tuts`.

File: hs/MakeElements.hs [hunk 2]
```diff
@@ -11,7 +11,7 @@ insertSVG :: H.AttributeValue -> H.Html
 insertSVG src = H.object ! A.data_ src ! A.type_ "image/svg+xml" $ ""
 
 createTag :: (H.Html -> H.Html) ->  H.AttributeValue -> H.AttributeValue -> H.Html -> H.Html
-createTag tag id class_ content = tag ! A.id id ! A.class_ class_ $ content
```
Comment (david-yz-liu) [inline hs/MakeElements.hs]: I think `id_` is more consistent here

File: hs/Svg/Parser.hs [hunk 3]
```diff
@@ -219,14 +219,14 @@ styles attrs =
...
-        toStyle s =
-            case splitOn ":" s of
+        toStyle split =
+            case splitOn ":" split of
             [n,v] -> (n,v)
             _ -> ("","")
 
 -- | Gets a style attribute from a style string.
 styleVal :: String -> [(String, String)] -> String
-styleVal name styles = fromMaybe "" $ lookup name styles
```
Comment (david-yz-liu) [inline hs/Svg/Parser.hs]: Again, the pluralization is good. Perhaps `styleMap`

**Comment by cchens** [comment]
@david-yz-liu Thanks, I've changed the names you've mentioned.

**Comment by david-yz-liu** [comment]
Thanks, @cchens!

**Pull request merged**